packages/shorebird_cli/test/src/commands/release/release_ios_framework_command_test.dart

```diff
@@ -447,5 +447,60 @@ flutter:
       ).called(1);
       expect(exitCode, ExitCode.success.code);
     });
+
+    test(
+        'succeeds when release is successful '
+        'with flavors and target', () async {
+      const flavor = 'development';
+      final target = p.join('lib', 'main_development.dart');
+      when(() => argResults['flavor']).thenReturn(flavor);
+      when(() => argResults['target']).thenReturn(target);
```

Comment:
We probably want to verify that these were provided as arguments to the flutter command and/or flavor text is present in the success output

Reply:
yeah good call we need to update this in other places as well. Will do in a separate PR 👍